kaos/models/models.py

```diff
@@ -0,0 +1,233 @@
+
+    uid = DB.Column(DB.Integer, primary_key=True)
+    response = DB.Column(DB.String, nullable=False)
+
+    def __init__(self, response):
+        self.response = response
+
+    def __repr__(self):
+        return '<Response History: uid={}>'.format(self.uid)
+
+    @classmethod
+    def __declare_last__(cls):
+        ValidateInteger(ResponseHistory.uid)
+        ValidateString(ResponseHistory.response)
+
+    @staticmethod
+    def get_by_id(uid):
+        """Find a response history entry by the UID assigned to the request when it was first made.
+
+        Args:
+            uid: The unique ID for the transaction.
+
+        Returns:
+            The ResponseHistory object, or None if not found.
+        """
+        return Satellite.query.get(uid)
+
+
+class OrbitSegment(SavableModel, DB.Model):
+    """This table stores satellite ephemeris segment records which are collections of OrbitRecords
+    in a given time period. These records are grouped together since some calculations must not use
+    data that crosses segment boundaries.
+
+    The table holds the following information:
+        segment_id:     Unique ID for a particular time segment
+        platform_id:    Unique ID for the satellite that owns this segment
+        start_time:     Time in seconds since the Linux epoch that records in this segment start on
+        end_time:       Time in seconds since the Linux epoch that records in this segment end on
+        orbit_records:  Satellite ephemeris records that fall within the time segment and are owned
+                        by the platform_id
+    """
+    __tablename__ = "OrbitSegment"
+
+    segment_id = DB.Column(DB.Integer, primary_key=True)
+    platform_id = DB.Column(DB.Integer, DB.ForeignKey('Satellite.platform_id'), nullable=False)
+    start_time = DB.Column(DB.Float, nullable=False)
+    end_time = DB.Column(DB.Float, nullable=False)
+    orbit_records = DB.relationship("OrbitRecord", backref='orbit_segment', lazy=True)
+
+    # Add an index to improve query time for get_by_platform_and_time
+    __table_args__ = (Index('OrbitSegment__platform_id__start_time', 'platform_id', 'start_time'), )
+
+    @classmethod
+    def __declare_last__(cls):
+        ValidateInteger(OrbitSegment.segment_id)
+        ValidateInteger(OrbitSegment.platform_id)
+
+    @staticmethod
+    def get_by_platform_and_time(platform_id, timestamp):
+        """Find the segment that contains the given time, for the specified satellite.
+
+        Args:
+            platform_id: The unique ID of the satellite.
+            timestamp: The time in seconds since the Unix epoch.
+
+        Returns:
+            The segment with a matching platform_id, and where start_time <= timestamp <= end_time.
+            If more than one segment matches, which occurs only if the timestamp is the boundary
+            of two segments, return the later segment. If no segment matches, return None.
+        """
+        # pylint: disable=undefined-variable
```

Comment:
Are we trying to remove all these comments?

Reply:
Nope, just merging 2 sections that both had pylint-disables